dashboard/app/models/course.rb

```diff
@@ -252,14 +255,10 @@ def self.has_any_course_experiments?(user)
   # Get the set of valid courses for the dropdown in our sections table, using
   # any alternate scripts based on any experiments the user belongs to.
   def self.valid_courses_without_cache(user: nil)
-    course_infos = Course.
-      where(name: ScriptConstants::CATEGORIES[:full_course]).
+    course_infos = Course.all.
+      select {|course| course.visible? && course.stable?}.
```

Comment:
this doesn't look right -- after soft launch, we'll need all visible courses, including those which are not stable, so that they still appear in the assignment selector, as "under development".

Reply:
I think I got confused by the comment below that line (which I deleted in this change). Would it be correct if the condition was just `course.visible?`?

Reply:
that comment you removed is confusing! yes, I think we just want to decide based on `course.visible?`.

Reply:
agree that course.visible? seems right

Reply:
I saw your request to re-review, but also still see `course.stable?` as part of this predicate which looks like it still needs to be removed. apologies if this is a repeat, I know I wrote this same comment in the last 24 hours but I don't see where it went in the PR